pyproject.toml

```diff
@@ -5,11 +5,71 @@ requires = [
   "setuptools_scm[toml]>=7.0",
   "wheel",
   "oldest-supported-numpy",
-  "Cython>=3.0"
+  "Cython>=3.0",
 ]
 # Defined by PEP 517
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "cf-units"
+authors = [
+    {name = "cf-units Contributors", email = "scitools.pub@gmail.com"}
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Atmospheric Science",
+    "Topic :: Scientific/Engineering :: Oceanography",
+]
+dependencies = [
+    "cftime >=1.2",
+    "jinja2",
+    "numpy ==1.26.4"
```

Comment:
Why is this pinned to exact number, again ?
I thought we got rid of that in #470 
( [here](https://github.com/SciTools/cf-units/pull/470/files#r1777145927) )

Reply:
Ooops! Great catch

e34a68ece77ecc39c946933c1bbf14916317dbfb